docs/src/internals.md

```diff
@@ -568,40 +578,110 @@ a formula with a call to `poly`:
 
-julia> sim_dat[:y] = randn(100) .+ 1 .+ 2*sim_dat[:b] .+ 3*sim_dat[:b].^2;
+julia> sim_dat.y = randn(100) .+ 1 .+ 2*sim_dat.b .+ 3*sim_dat.b.^2;
 
 julia> fit(LinearModel, @formula(y ~ 1 + poly(b,2)), sim_dat)
-StatsModels.TableRegressionModel{LinearModel{LmResp{Array{Float64,1}},DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}
+StatsModels.TableRegressionModel{LinearModel{GLM.LmResp{Array{Float64,1}},GLM.DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}
 
 y ~ 1 + :(poly(b, 2))
 
 Coefficients:
-────────────────────────────────────────────────────
-             Estimate  Std.Error   t value  Pr(>|t|)
-────────────────────────────────────────────────────
-(Intercept)  0.911363   0.310486   2.93528    0.0042
-poly(b, 2)   2.94442    0.191024  15.4139     <1e-27
-────────────────────────────────────────────────────
+───────────────────────────────────────────────────────────────────────────
+             Estimate  Std. Error   t value  Pr(>|t|)  Lower 95%  Upper 95%
+───────────────────────────────────────────────────────────────────────────
+(Intercept)  0.911363    0.310486   2.93528    0.0042   0.295214    1.52751
+poly(b, 2)   2.94442     0.191024  15.4139     <1e-27   2.56534     3.3235
+───────────────────────────────────────────────────────────────────────────
 
 julia> fit(GeneralizedLinearModel, @formula(y ~ 1 + poly(b,2)), sim_dat, Normal())
-StatsModels.TableRegressionModel{GeneralizedLinearModel{GlmResp{Array{Float64,1},Normal{Float64},IdentityLink},DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}
+StatsModels.TableRegressionModel{GeneralizedLinearModel{GLM.GlmResp{Array{Float64,1},Normal{Float64},IdentityLink},GLM.DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}
 
 y ~ 1 + poly(b, 2)
 
 Coefficients:
-───────────────────────────────────────────────────
-             Estimate  Std.Error  z value  Pr(>|z|)
-───────────────────────────────────────────────────
-(Intercept)  0.829374  0.131582    6.3031    <1e-9
-b^1          2.13096   0.100552   21.1926    <1e-98
-b^2          3.1132    0.0813107  38.2877    <1e-99
-───────────────────────────────────────────────────
+──────────────────────────────────────────────────────────────────────────
+             Estimate  Std. Error  z value  Pr(>|z|)  Lower 95%  Upper 95%
+──────────────────────────────────────────────────────────────────────────
+(Intercept)  0.829374   0.131582    6.3031    <1e-9    0.571478    1.08727
+b^1          2.13096    0.100552   21.1926    <1e-98   1.93388     2.32804
+b^2          3.1132     0.0813107  38.2877    <1e-99   2.95384     3.27257
+──────────────────────────────────────────────────────────────────────────
 
 ```
 
 (a `GeneralizeLinearModel` with a `Normal` distribution is equivalent to a
 `LinearModel`)
 
+### [Making special syntax "runtime friendly"] (@id extend-runtime)
```

Comment:
Should this section be above the last?
It seems more closely re.ated to the example

Reply:
Everything in this section is about the same example, and the thing about the context is really important (more so than supporting run-time creation I'd say).  Then again, I'm not totally committed to this order.  I'd just written the earlier parts in one go and didn't want to have to edit those sections but I can have another look.